modules/web-console/frontend/app/components/page-queries/components/queries-notebook/controller.ts

```diff
@@ -1587,6 +1588,8 @@ export class NotebookCtrl {
                                 collocated
                             };
 
+                            console.log(qryArg);
```

Comment:
Please remove.

Reply:
@Klaster1 Removed

Reply:
Don't forget to remove lodash imports, if no longer needed.